gapis/perfetto/android/trace.go

```diff
@@ -189,6 +190,12 @@ func Start(ctx context.Context, d adb.Device, a *android.ActivityAction, opts *s
 		return nil, cleanup.Invoke(ctx), err
 	}
 
+	// Remove any left over trace files, as, if tracing to a file on the device is
+	// requested, Perfetto would fail if the file already exists.
+	if err := d.RemoveFile(ctx, perfettoTraceFile); err != nil {
+		log.E(ctx, "Failed to delete previous perfetto trace file %v", err)
```

Comment:
I was thinking about that and decided to just let it continue. If the error during the delete happened because the file actually wasn't deleted, Perfetto will fail right away just a bit later and the error message is good and the user can act on it by removing the file. However, if we fail here for some other reason, or incorrectly think we failed, we would be aborting the trace with a strange error that the user probably can't do anything about.